src/_slugline_vars.scss

```diff
@@ -2,9 +2,7 @@
 
 $primary: #f2c0d9;
 
-// Use our custom fonts
-$body-ff: "Joanna Nova", serif;
-$headings-ff: "DIN Next", sans-serif;
+@import "slugline_fonts";
```

Comment:
This import (and any SCSS that generates a CSS rule), needs to be moved to slugline.scss, or any component that imports this variable file will create a copy of the `font-face` selectors.

slugline.scss is imported only once, so any top-level rules should go there.